packages/e2e-test-utils/src/login-user.js

```diff
@@ -27,8 +29,9 @@ export async function loginUser(
 	await pressKeyWithModifier( 'primary', 'a' );
 	await page.type( '#user_pass', password );
 
+	const waitForLoginNavigation = page.waitForNavigation();
 	await Promise.all( [
-		page.waitForNavigation(),
+		waitForLoginNavigation,
 		page.click( '#wp-submit' ),
 	] );
```

Comment:
This should change to this in order for the tests to stop failing:
```js
	await Promise.all( [ waitForLoginNavigation, page.click( '#wp-submit' ) ] );
```

Reply:
I've committed the change, and the linting task appears to pass now 🎉